cmd/crictl/image.go

```diff
@@ -182,15 +186,19 @@ var listImageCommand = &cli.Command{
 		// output in table format by default.
 		display := newTableDisplay(20, 1, 3, ' ', 0)
 		verbose := c.Bool("verbose")
+		showPinned := c.Bool("pinned")
 		showDigest := c.Bool("digests")
 		quiet := c.Bool("quiet")
 		noTrunc := c.Bool("no-trunc")
 		if !verbose && !quiet {
+			row := []string{columnImage, columnTag}
+			if showPinned {
+				row = append(row, columnPinned)
```

Comment:
can we add a new column to the end? The column is not that important and in most cases will only be true for a single image

Reply:
My thinking is that when the user passes the --pinned flag, maybe he is trying to view the pinned information, so I put this column in front of it.
 if I put it at the end this is what it would look like, one more line of code:
```go
			row := []string{columnImage, columnTag}
			if showPinned {
				row = append(row, columnPinned)
			}
			row = append(row, columnImageID, columnSize)      
			if showDigest {
				row = append(row, columnDigest)
			}
			display.AddRow(row)
```

Of course, I'm keeping an open mind about the placement of this column, if you suggest putting it at the end , then I'll update this

Reply:
Trying to understand the history here. I think the `--digests` was added to match `docker cli`. And docker has it because there were no digests for older images: https://docs.docker.com/engine/reference/commandline/images/#digests

If we want to keep compatibility with `docker`, having an additional flag and not changing the default output makes sense.

It may be a good idea to create a test for this and explicitly mention that the ordering of columns and the default list of columns is driven by compatibility with the docker cli.

Ordering-wise I really don't like the idea of inserting column in the middle. If customer wants it - they can use template to specify the exact order of columns.

My suggestion will be to put pinned to be the last column.